### ISSUE TITLE
Adding loading cache kernel with clCreateProgramWithBinary

### ISSUE THREAD

**Issue by hksonngan** [hughperkins/EasyCL#6]
everty time compiling kernel in AMD and Nvidia speed is acceptable but in Intel it quite slow (my GPU Intel HD4000 compile cl file with 600 line it take over 2 minutes)
So we can save compiled kernel with clGetProgramInfo( _program, CL_PROGRAM_BINARIES, sizeof( unsigned char* ) * nDevices, pgBinaries, NULL ); and next time loading clCreateProgramWithBinary

**Comment by hughperkins** [comment]
Yes... but I dont have time to do that right now.  Is this something you might consider looking into?
